static/app/components/tabs/item.tsx

```diff
@@ -0,0 +1,12 @@
+import {Item as _Item} from '@react-stately/collections';
+import {ItemProps} from '@react-types/shared';
+import {LocationDescriptor} from 'history';
+
+export interface TabListItemProps extends ItemProps<any> {
```

Comment:
Pulled `Item` into its own file so that I could import these props in `tabList.tsx`

Reply:
Good idea!